.github/workflows/docker-build.yml

```diff
@@ -0,0 +1,33 @@
+name: Ensure Docker container for Ruby can build successfully 
+
+on:
+  push:
+    branches: [ "*" ]
+  pull_request:
+    branches: [ "*" ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Build Images
+        working-directory: ./
+        run: |
+           docker compose build
+           
+      - name: Run containers 
+        working-directory: ./
+        run: |
+          docker compose up -d
+  
+      - name: Wait for the database to be ready
+        run: sleep 60s
+
```

Comment:
this should not be necessary -- the docker-compose file itself contains health checks that will delay the server startup until the database is available.

Reply:
Please see https://github.com/rubyforgood/stocks-in-the-future/pull/26/commits/3fec5dc3067b967e999d1aa3d4a0bf91ebd5c3c8